src/RightsService.Data/UserRoleRepository.cs

```diff
@@ -102,19 +118,23 @@ public async Task<List<DbUserRole>> GetWithRightsAsync()
 
     public async Task<bool> RemoveAsync(Guid userId)
     {
-      DbUserRole user = await _provider.UsersRoles.FirstOrDefaultAsync(u => u.UserId == userId && u.IsActive);
+      DbUserRole user = await _provider.UsersRoles.FirstOrDefaultAsync(u => u.UserId == userId);
```

Comment:
may be return "&& u.IsActive"? this protects against re-writing to the history table if your method is called not from your command but from another place